association-de-science-regionale-de-langue-francaise.csl

```diff
@@ -0,0 +1,482 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="fr-CA">
+  <info>
+    <title>Association de Science Régionale de Langue Française (French)</title>
+    <title-short>ASRDLF</title-short>
+    <id>http://www.zotero.org/styles/association-de-science-regionale-de-langue-francaise</id>
+    <link href="http://www.zotero.org/styles/association-de-science-regionale-de-langue-francaise" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
+    <link href="http://asrdlf2016.com/sites/default/files/users/user167/feuille_de_style_asrdlf2016c.docx" rel="documentation"/>
+    <author>
+      <name>Florian Martin-Bariteau</name>
+      <email>f.martin-bariteau@umontreal.ca</email>
+      <uri>http://f-mb.org/</uri>
+    </author>
+    <contributor>
+      <name>Dominic Villeneuve</name>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="psychology"/>
+    <category field="generic-base"/>
```

Comment:
Can you pick better categories from the list at http://docs.citationstyles.org/en/stable/specification.html#appendix-i-categories? Maybe "geography" and "sociology"?